test-tools/test_state_recover.sh

```diff
@@ -177,38 +184,68 @@ function step2_delete_old_state()
     echo "Created signed state amount: $signed_state_amount"
 
     #random delete some states
-    random_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
+    state_delete_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
 
-    echo "Deleting $random_num signed states"
+    echo "Deleting $state_delete_num signed states"
 
     # remove most of states only leaving one
     for ((i=1;i<=random_num;i++)); do
         delete_last_round
     done 
 }
 
+function delete_event_files() {
+    print_banner "Running ${FUNCNAME[0]}"
+    event_files_amount=`ls -tr data/eventStream/events_0.0.3/*evts | sort -n| wc -l `
+    echo "There are $event_files_amount event files"
+    random_num=$(( ( RANDOM % ($state_delete_num ) * 2 ) + 2 ))
```

Comment:
Do you $event_files_amount here?

Reply:
number of events files generated is four time of number of signed state files generation

Assume "*" is the state file, "^" is event file 
after the first run
```
*__*__*__*__*__*__*
^^^^^^^^^^^^^^^^^^
```

After delete 2 state file

```
*__*__*__*__*
^^^^^^^^^^^^^^^^^^
```

Then we random delete some, lets say 5, event fils 

```
*__*__*__*__*
^^^^^^^^^^^^^
```
Then we still have more events files left (2 files), in terms of the last saved signed state files,
 to do recover test